modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.install

```diff
@@ -19,30 +19,37 @@ function oe_bootstrap_theme_paragraphs_install($is_syncing): void {
   if ($is_syncing) {
     return;
   }
-  $storage = new FileStorage(drupal_get_path('module', 'oe_bootstrap_theme_paragraphs') . '/config/overrides/');
-  $config_manager = \Drupal::service('config.manager');
-  $entity_type_manager = \Drupal::entityTypeManager();
 
-  $field_config = [
+  $folder = '/config/overrides/';
```

Comment:
$folder is used only below, so no need to define it as a variable, remove this line